src/react-components/info-dialog.js

```diff
@@ -111,7 +111,8 @@ class InfoDialog extends Component {
             <a href="https://webvr-slack.herokuapp.com/" target="_blank" rel="noopener noreferrer">
               WebVR Slack
             </a>{" "}
-            in the #social channel.<br />VR meetups every Friday at noon PST!
+            in the <a href="https://webvr.slack.com/messages/social" target="_blank" rel="noopener noreferrer">#social</a> channel.<br />
+            VR meetups every Friday at noon PDT!
```

Comment:
@gfodor: notice `PST` -> `PDT` (do you want to just say `Pacific Time`?)

Reply:
ah yeah that seems right to me!